packages/runtime/datastore/src/dataStoreRuntime.ts

```diff
@@ -446,12 +446,12 @@ IFluidDataStoreChannel, IFluidDataStoreRuntime, IFluidHandleContext {
      * 1. Sending an Attach op that includes all existing state
      * 2. Attaching the graph if the data store becomes attached.
      */
-    public bindToContext() {
```

Comment:
`bindToContext` should be removed and the contents should be moved to `makeVisibleAndAttachGraph`. This will ensure that existing calls to `bindToContext` by casting to any would not work. `bindState` will also be no longer needed.

Reply:
+1

Reply:
I went ahead and removed BindState enum too then.